src/mixins/amalgamation-mixin.ts

```diff
@@ -221,9 +219,9 @@ export default class AmalgamationMixin extends Vue {
           role: AmlRoles.AMALGAMATING,
           identifier: tingBusiness.businessInfo.identifier,
           name: tingBusiness.businessInfo.legalName,
-          email: tingBusiness.authInfo.contacts[0].email,
+          email: tingBusiness.authInfo?.contacts[0].email,
```

Comment:
If unaffiliated, the optional chaining deals with this and the address on line 224. This is since if it's unaffiliated, both are undefined. So, these will be not set.

If affiliated, things would go normal.